lang/en/auth_saml2.php

```diff
@@ -108,6 +108,16 @@
 $string['manageidpsheading'] = 'Manage available IdPs';
 $string['mdlattr_help'] = 'Which Moodle user field should the IdP attribute be matched to?';
 $string['mdlattr'] = 'Mapping Moodle';
+$string['wantassertionssigned'] = 'Want Assertions Signed';
+$string['wantassertionssigned_help'] = '';
+$string['assertionsconsumerservices'] = 'Assertions Consumer Services';
+$string['assertionsconsumerservices_help'] = '';
+$string['entityid'] = 'Entity ID';
+$string['entityid_help'] = '';
+$string['allowcreate'] = 'Allow Create';
+$string['allowcreate_help'] = '';
```

Comment:
This definitely needs docs to tell it apart from 'autocreate' and I think both the autocreate and these docs should be refined something like:

allowcreate Allow creation of IdP users on demand
autocreate Allow creation of Moodle users on demand